doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_flpr.rst

```diff
@@ -0,0 +1,178 @@
+.. _ug_nrf54h20_flpr:
+
+Working with the FLPR core
+##########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+.. note::
+   The FLPR core support in the |NCS| is currently :ref:`experimental<software_maturity>`.
+
+The nRF54H20 SoC includes a dedicated VPR CPU, based on RISC-V architecture, known as the *fast lightweight peripheral processor* (FLPR).
+The FLPR core can be used to manage specific peripherals through the appropriate Zephyr Device Driver API:
+
+* UARTE120
+
+##List to be completed
+
+.. _vpr_flpr_nrf54h20_initiating:
+
+Using Zephyr multithreaded mode on FLPR
+***************************************
+
+The FLPR core can operate as a general-purpose core, running under the full Zephyr kernel.
+To enable the FLPR core, always include an overlay in the application core build that configures and builds the FLPR target.
+
+Bootstrapping the FLPR core
+===========================
+
+The |NCS| provides a FLPR snippet that adds the overlay needed for bootstrapping the FLPR core.
+The primary purpose of this snippet is to enable the transfer of the FLPR code to the designated memory region (if required) and initiate the FLPR core.
+
+When building for the ``nrf54h20dk/nrf54h20/cpuflpr`` target, a minimal sample is automatically loaded onto the application core.
+For more details, see :ref:`building_nrf54h_app_flpr_core`.
+
+Memory allocation
+*****************
+
+Running the FLPR CPU can lead to increased latency when accessing ``RAM_21``.
+To mitigate this, you should use ``RAM_21`` exclusively for FLPR code, FLPR data, and non-time-sensitive information from the application CPU.
+For data that requires strict access times, such as CPU data used in low-latency ISRs, you should use local RAM or, when greater latency is acceptable, ``RAM_0x``.
+The DMA buffers should be placed in memory designed to a given peripheral.
+
+.. _building_nrf54h:
+
+Building and programming with the nRF54H20 DK
+*********************************************
+
+.. note::
+   The FLPR core support in the |NCS| is currently :ref:`experimental<software_maturity>`.
+
+For samples targeting the FLPR core, program the application core, which subsequently programs the FLPR core.
```

Comment:
Not true?

Reply:
Shantha asked for this edit @shanthanordic @masz-nordic  Could you please clarify this?

Reply:
Reverted